### ISSUE TITLE
Card Info Separation

### ISSUE THREAD

**Issue by Ledge-And-Dairy** [Cockatrice/Cockatrice#1733]
I was wondering if we could seperate these two things so that if you have the "both" option in the deck builder section you dont open up a game it is also forced onto the "both" option and vise versa for when you launch Cockatrice after using "Image"/ "Info" from last time you were in a game.

https://gyazo.com/3f3d51e086c821227627e61a9efbdbae - when i set it to both in game.
https://gyazo.com/e4ed8ed4417e5be6109cd17d3863f6b8 - when i then proceed to open a game afterwards.

**Comment by tooomm** [comment]
would be a nice addition :+1: 
one state for deck editor and another one for game rooms

**Comment by ZeldaZach** [comment]
Completed already

**Comment by tooomm** [comment]
Nope, the mode selected for displaying card information in the `Card Info` widget in `Deck Editor` is wired to the one in a game and both can't display different modes (pic/text/both).

**Comment by Daenyth** [comment]
derp

**Comment by ebbit1q** [comment]
I think this widget is the same object on both of these, so it'd require some separating to puzzle out
